warehouse/query-core/src/main/java/datawave/core/iterators/CompositeSeekingIterator.java

```diff
@@ -74,7 +75,7 @@ public void init(SortedKeyValueIterator<Key,Value> source, Map<String,String> op
             if (typeClass != null) {
                 try {
                     type = Class.forName(typeClass).asSubclass(DiscreteIndexType.class).getDeclaredConstructor().newInstance();
-                } catch (Exception e) {
+                } catch (ClassNotFoundException | NoSuchMethodException | InstantiationException | IllegalAccessException | InvocationTargetException e) {
                     log.warn("Unable to create DiscreteIndexType for class name: " + typeClass);
```

Comment:
Is it worth logging out e or e's message here?

Reply:
+1